Libraries/LibJS/Tests/builtins/Date/Date.parse.js

```diff
@@ -24,6 +24,7 @@ test("basic functionality", () => {
     expect(Date.parse("2021-07-01 03:00Z")).toBe(1625108400000);
     expect(Date.parse("2024-01-08 9:00Z")).toBe(1704704400000);
     expect(Date.parse("Wed, 17 Jan 2024 11:36:34 +0000")).toBe(1705491394000);
+    expect(Data.parse("Thu, 09 Jan 2025 23:00:00")).toBe(1736460000000);
```

Comment:
Did you run this test? There's a typo, `Data` instead of `Date`...

Reply:
Didn't find the exact command line to run only Date tests, and running all tests would produce lots of errors locally. So I didn't ran the test :/ I pushed a fix.

Reply:
> running all tests would produce lots of errors locally

This should definitely not be the case! Care to discuss further on Discord?

Reply:
Not sure what went wrong, but I deleted Build folder, and ran tests again and now it's fine. I'll send a message in Discord if I ever stumble upon a problem.